package.json

```diff
@@ -1,21 +1,33 @@
 {
-  "name": "sumo-ot-js",
+  "name": "sumo-opentelemetry",
   "version": "0.0.0",
-  "private": true,
+  "description": "Sumo Logic all-in-one library for OpenTelemetry tracing auto-instrumentation in web",
+  "main": "./dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "build": "node ./scripts/bundle.js && rollup -c"
   },
-  "dependencies": {
-    "@types/shimmer": "^1.0.1",
-    "shimmer": "^1.2.1",
-    "zone.js": "^0.10.3"
-  },
+  "repository": "SumoLogic/sumologic-opentelemetry-js",
+  "author": "Sumo Logic Authors",
+  "license": "Apache-2.0",
```

Comment:
I assume this was consulted with our legal team?

Reply:
That's the license we use for other projects, etc. Kubernetes Collection:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/LICENSE

Reply:
We have our NPM account. Credentials to it are stored in 1Password.

Reply:
`LICENSE` file added.